apps/docs/content/components/Hero/react.mdx

```diff
@@ -43,6 +43,37 @@ import {Hero} from '@primer/react-brand'
 </Hero>
 ```
 
+### Images
+
+`Hero` can display an optional foreground image. Use `imagePosition` to alternate between various layouts.
+
+```jsx live
+<>
+  <Hero>
+    <Hero.Label>Actions</Hero.Label>
+    <Hero.Heading>Automate your workflow from idea to production</Hero.Heading>
+    <Hero.PrimaryAction href="#">Get started with Actions</Hero.PrimaryAction>
+    <Hero.Image
+      src="https://via.placeholder.com/600x400/d3d9df/d3d9df.png"
+      alt="placeholder, blank area with an off-white background color"
+    />
+  </Hero>
+  <br />
+  <Hero align="center">
+    <Hero.Label>Actions</Hero.Label>
+    <Hero.Heading size="2">
+      Automate your workflow from idea to production
+    </Hero.Heading>
+    <Hero.PrimaryAction href="#">Get started with Actions</Hero.PrimaryAction>
+    <Hero.Image
+      imagePosition="inline-end"
```

Comment:
shouldn't we add a prop to the hero parent to manage this instead? It feels strange that a subitem prop changes the layout of the component

Reply:
It's because the image is optional and `imagePosition` only makes sense if you're passing a child called `Hero.Image`, hence the coupling. 

I agree it's a bit of indirection in the parent, but the user won't see that. The trade-off I'm thinking about here is that the user wants to insert an image, so the configurability options for that _image_ is co-located to the child. It's a devexp feature more than anything.

Does that help contextualise it, or do you feel strongly about it moving into the parent?

Reply:
thanks for the context, not strong thoughts on this.